packages/sanity/src/core/components/hookCollection/GetHookCollectionState.tsx

```diff
@@ -13,13 +13,14 @@ export interface GetHookCollectionStateProps<T, K> {
   children: (props: {states: K[]}) => ReactNode
   hooks: ActionHook<T, K>[]
   onReset?: () => void
+  group?: string
```

Comment:
Even though it's internal, could we add a comment explaining it's purpose and what we currently use it for?